filter_plugins/openshift_node.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Custom filters for use in openshift-node
+'''
+from ansible import errors
+
+class FilterModule(object):
+    ''' Custom ansible filters for use by openshift_node role'''
+    
+    @staticmethod
+    def get_dns_ip(openshift_dns_ip, hostvars):
```

Comment:
@abutcher mind checking that this accurately replicates the what was previously in ansible?